test/e2e/curated_packages_test.go

```diff
@@ -273,3 +273,51 @@ func TestCPackagesVSphereKubernetes122BottleRocketSimpleFlow(t *testing.T) {
 	)
 	runCuratedPackageInstallSimpleFlow(test)
 }
+
+func TestCPackagesNutanixKubernetes120SimpleFlow(t *testing.T) {
+	test := framework.NewClusterE2ETest(t,
+		framework.NewNutanix(t),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube120)),
+		framework.WithPackageConfig(t, packageBundleURI(v1alpha1.Kube120),
+			eksaPackageControllerHelmChartName, eksaPackageControllerHelmURI,
+			eksaPackageControllerHelmVersion, eksaPackageControllerHelmValues),
+		framework.WithEnvVar("NUTANIX_PROVIDER", "true"),
+	)
+	runCuratedPackageInstallSimpleFlow(test)
+}
+
+func TestCPackagesNutanixKubernetes121SimpleFlow(t *testing.T) {
+	test := framework.NewClusterE2ETest(t,
+		framework.NewNutanix(t),
```

Comment:
Just to make sure you've considered it, if you wanted to test other host image OSes (I assume this defaults to Bottlerocket), that might be worthwhile. We don't officially support anything else right now that I'm aware of, but it seems like RedHat is around the corner, and I imagine Ubuntu will reappear at some later date.

Reply:
@ewollesen, we do have a ubuntu image for nutanix https://github.com/aws/eks-anywhere-build-tooling/pull/1332. 
for local testing, i was using .env file with appropriate image name as in 
<pre>
export T_NUTANIX_MACHINE_TEMPLATE_IMAGE_NAME=eksa-dm-image-ubuntu-2004-kube-v1.20
</pre>. 
How do you propose setting for CI run.

Reply:
@abhinavmpandey08 provided the link to the following file to update. https://github.com/aws/eks-anywhere/blob/main/cmd/integration_test/build/buildspecs/test-eks-a-cli.yml